scripts/Dockerfile-sync-check

```diff
@@ -1,12 +1,27 @@
 FROM fedora:36
+
 LABEL org.opencontainers.image.source https://github.com/chainsafe/forest-iac
 
 # Install dependencies
-RUN dnf install -y docker docker-compose ruby ruby-devel make gcc
-RUN gem install docker-api slack-ruby-client sys-filesystem
+RUN dnf -y install \
+    docker \
+    docker-compose \
+    ruby \
+    ruby-devel \
+    make \
+    gcc \
+    && dnf clean all
 
-WORKDIR /chainsafe
+# Gem install dependencies
+RUN gem install \
+    docker-api \
+    slack-ruby-client \
+    sys-filesystem
 
+# Set a working directory and copy files
```

Comment:
Are those comments really helpful?

Reply:
Well, I just felt it'd be good to make any reader understand the operations of each line.

Reply:
We should assume that all readers of a Dockerfile should know those basic keywords. Otherwise, we'll end up in a grotesque situation like this:
```
# change to the parent directory
cd ..
# print "finished" to the standard output
echo "finished"
# exit the script with code zero, which indicates a success
exit 0
```

I'm okay with documenting non-trivial or domain-specific code. But the comments should explain things, not reword the code.

For example:
```
# Gem install dependencies
RUN gem install <dependencies>
```
You are installing some dependencies. It's obvious. What we can add more to it, for those who never did any Ruby (a reasonable assumption that most people didn't)
```
# Install dependencies for the Ruby script
RUN gem install <dependencies>
```
This gives a link between `gem` and Ruby to the reader.